src/pages/RealmScans/main.ts

```diff
@@ -0,0 +1,75 @@
+import { pageInterface } from '../pageInterface';
+
+export const RealmScans: pageInterface = {
+  name: 'RealmScans',
+  domain: 'https://realmscans.com',
+  languages: ['English'],
+  type: 'manga',
+  isSyncPage(url) {
+    return j.$('#screenmode').length > 0;
+  },
+  isOverviewPage(url) {
+    return utils.urlPart(url, 3) === 'series';
```

Comment:
Overview gets triggered on the series list page:
https://realmscans.com/series/

adding `&& utils.urlPart(url, 4)` probably fixes it